contents/blog/posthog-vs-hotjar.md

```diff
@@ -0,0 +1,301 @@
+            <td>Roll out features safely; toggle features for cohorts or individuals</td>
+        </tr>
+        <tr>
+            <td><strong>A/B testing</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Test changes and analyze impact</td>
+        </tr>
+        <tr>
+            <td><strong>Feedback widgets</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Suggestion box on your site where users can express frustration or delight.</td>
+        </tr>
+        <tr>
+            <td><strong>In-app prompts and messages</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Send messages to users in your app</td>
+        </tr>
+        <tr>
+            <td><strong>Surveys</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Capture user feedback with surveys. </td>
+        </tr>
+        <atr>
+            <td><strong>Customer interviews</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Recruit and schedule customer interviews. </td>
+        </tr>
+        <tr>
+            <td><strong>Apps/Integrations</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Push and pull data to other destinations</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+- **Session replays:** [Session replays](/session-replay) in PostHog recreate exactly what real users see and how they use your product. They also enable you to debug problems using built-in console logs and network performance.
+
+- **Feature flags:** PostHog includes [multivariate feature flags](/feature-flags) that support JSON payloads, In-app prompts. Teams can use feature flags to offer different features or UI choices to users, to trigger in-app messages, and more.
+
+- **A/B testing:** In PostHog, you can use the [experimentation suite](/ab-testing) to create multivariate tests within your product, such as showing some users a different page layout to others. Over time, you can build an understanding of which page performs better, correlate results with other events, and deploy a final version.
+
```

Comment:
We could use the Tutorial Tracks component here to add some visual elements and further links to the page, as in the Pendo article: 
```
<h4 className="mb-0">Discover what's possible with <span className="text-blue">product analytics</span></h4>
<TutorialsSlider slugs={[
  "/tutorials/churn-rate",
  "/tutorials/funnels",
  "/tutorials/api-get-insights-persons",
  "/tutorials/feature-retention",
  "/tutorials/next-steps-after-installing"
]} />
```